packages/react/src/file-upload/FileUploadExistingFileList.stories.tsx

```diff
@@ -25,6 +25,21 @@ export const Basic: Story = {
 				size: 123456,
 				thumbnailSrc: 'https://placehold.co/144',
 			},
+			{
+				name: 'example.jpg',
+				size: 123456,
+				thumbnailSrc: 'https://placehold.co/144',
+				href: '#',
+			},
+			{
+				name: 'onclick.jpg',
+				size: 123456,
+				href: '#',
+				thumbnailSrc: 'https://placehold.co/144',
+				onClick: () => {
+					console.log('Existing file clicked');
+				},
```

Comment:
I'd do this it doesn't open a new tab
```suggestion
				onClick: (event) => {
					event.preventDefault();
					console.log('Existing file clicked');
				},
```